src/d_main.c

```diff
@@ -1203,7 +1276,8 @@ static int GuessFileType(const char *name)
         iwad_found = true;
     }
     else if (M_StringEndsWith(lower, ".wad") ||
-             M_StringEndsWith(lower, ".lmp"))
+             M_StringEndsWith(lower, ".lmp") ||
+             M_StringEndsWith(lower, ".zip"))
```

Comment:
Could you also add zip to the bash completion file?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/649/commits/eae2bc0e9b31ea7d30e8b197ce038e6dbe8e96b1